data/json/achievements.json

```diff
@@ -11,5 +11,25 @@
     "description": "Rude awakening",
     "time_constraint": { "since": "game_start", "is": "<=", "target": "1 minute" },
     "requirements": [ { "event_statistic": "num_avatar_kills", "is": ">=", "target": 1 } ]
+  },
+  {
+    "id": "achievement_marathon",
+    "type": "achievement",
+    "description": "Pheidippides was a hack",
+    "//": "Run a marathon...plus a little bit more.",
+    "requirements": [ { "event_statistic": "num_moves_ran", "is": ">=", "target": 42196 } ]
+  },
+  {
+    "id": "achievement_proclaimers",
```

Comment:
I'd give this a name closer to the definition rather than the inspiration, like `achievement_walk_1000_miles`.  Just in case it ever changes description.

Reply:
Fine by me.